### DISCUSSION TITLE
shen-ping/info/AxlxwO8O9SXUQPBlWhcjV

### DISCUSSION THREAD

**Discussion by giscus** [wurencaideli/dumogu-ypwz-comment#17] [Announcements]
# shen-ping/info/AxlxwO8O9SXUQPBlWhcjV

不是只有视频和音乐才带给人欢乐！还有一篇文字哟！学生时代所看过的短文还有谁记得呢？甚是怀念呢！

https://www.dumogu.top/shen-ping/info/AxlxwO8O9SXUQPBlWhcjV?isInfo=false

**Comment by wurencaideli** [comment]
当年的qq看点现在也无了，可惜了这么多段子手
